src/main/java/io/cdap/e2e/pages/locators/CdfStudioLocators.java

```diff
@@ -265,4 +265,26 @@ public static By locatorOfLoadingSpinnerOnValidateButton() {
   @Deprecated
   @FindBy(how = How.XPATH, using = "//*[@data-cy='plugin-validation-error-msg']")
   public static WebElement pluginValidationErrorMsg;
+
+  @FindBy(how = How.XPATH, using = "//img[@id='resource-center-btn']")
+  public static WebElement uploadFileButton;
```

Comment:
The naming for this is confusing, the methods that ultimately uses this button are called clickOnPlusGreenButton --> clickOnAddEntityButton. Should rename to addEntityButton? Because the actual file upload to import the pipeline happens in the next step.

Reply:
Resolved.